test/mocks/lock.js

```diff
@@ -0,0 +1,42 @@
+var Fiber = require('fibers');
+
+var locked = false;
```

Comment:
Only one lock can be created at a time since this is a global, right?
Can you add a short file comment describing this file?

Reply:
That's how the LockService works. Added a note.